decoder/README.md

```diff
@@ -0,0 +1,6 @@
+This package defines [`IEvidinceDecoder`](ievidecoder.go) and
+[`IEndorsmentDecoder`](iendorsementdocoder.go) [pluggable](../plugin/README.md)
```

Comment:
```suggestion
This package defines [`IEvidenceDecoder`](ievidencedecoder.go) and
[`IEndorsementDecoder`](iendorsementdocoder.go) [pluggable](../plugin/README.md)
```

Reply:
> > Note: as previsouly discussed, there is one outstanding issue with the current refactior -- then naming of the VTS-side plugin interface. It has been renamed to `IEvidenceDecoder` since the name "scheme" isn't appropriate, given that both plugins can be equality thought to be part o fthe scheme. However, as noted, "evidence decoder" may not be the best name given the scop of its duties, however I'm having a hard time thinking of a better alternative. Related to this is the fact that it's name (accessed via `GetName` is referred to as a "scheme" by the endorsement decoder; this is not accurate (the "Scheme" in `Endorsement` struct refers to this plugin, not to the endorsement scheme).

Should we call IEvidenceDecoder to IEvidenceProcessor or IEvidenceHandler. They are atleast more generic then pure Decoding and renders a bit more clarity. WDYT ?

Reply:
Yes, "handler", while still not perfect, makes more sense than "decoder". I'll make the change.